fetch_api/src/fetch_api/arm.py

```diff
@@ -76,6 +76,19 @@ def __init__(self):
         self._move_group_client.wait_for_server(rospy.Duration(10))
         self._compute_ik = rospy.ServiceProxy('compute_ik', GetPositionIK)
         self._tf_listener = TransformListener()
+        self.move_group = MoveGroupInterface(ARM_GROUP_NAME, "base_link")
+
+    """
```

Comment:
Python docstrings go right under the signature

Reply:
A summary of how this behaves would also be helpful. Does it check for collisions? What happens if there's no safe way to tuck?